openedx/features/badging/tests/test_views.py

```diff
@@ -11,10 +14,23 @@
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from xmodule.modulestore.tests.factories import CourseFactory
 
-from .factories import BadgeFactory, UserBadgeFactory
+from nodebb.constants import CONVERSATIONALIST_ENTRY_INDEX, TEAM_PLAYER_ENTRY_INDEX
+
+from .factories import BadgeFactory, UserBadgeFactory, DiscussionCommunityFactory
 from .. import views as badging_views
+from ..constants import (
+    COURSES_KEY,
+    CONVERSATIONALIST,
+    DISCUSSION_ID_KEY,
+    DISCUSSION_COUNT_KEY,
+    USERNAME_KEY
+)
+
+STATUS_CODE_TRUE = 200
+STATUS_CODE_FALSE = 400
```

Comment:
We can user following instead

`from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST`